### DISCUSSION TITLE
Pas de réponse ddt4all

### DISCUSSION THREAD

**Discussion by julien133** [cedricp/ddt4all#1098] [Q&A] [answered]
Bonjour à tous,

Je me suis récemment inscrit ici dans le but d'utiliser le logiciel ddt4all pour résoudre le problème de dilution d'huile sur ma clio 3 dci.

Je suis sous Windows 11 64, j'ai donc installé la dernière version de ddt4all :  DTT4ALL-Windows-Installer-v3.0.1_Stable-x64.exe.
Pour la connection avec la voiture, j'utilise un ELM ODB 327 wifi v1.5 (le bleu et orange qu'on trouve partout).

J'ai directement eu le problème suivant : 

![error](https://github.com/cedricp/ddt4all/assets/168999811/162d9c5c-b4a6-4318-8125-09e6ea02f8d2)

En cherchant dans le forum j'ai trouvé qu'en modifiant une ligne dans le fichier elm.py cela résoudrait ce problème : 

![elm modif](https://github.com/cedricp/ddt4all/assets/168999811/0572946a-8ada-4872-8317-29dccf7ff5f2)

Cela a ainsi permis de se connecter à la voiture.

J'ai ensuite chargé le DCM34 (ECM de la clio 3), jusque là pas de soucis même si ça rame beaucoup.
Ensuite je charge le module IVP et là ça rame énormément, cela met plusieurs minutes avant de m'afficher l'écran suivant, dont je n'arrive pas à lire les valeurs : 

![no data](https://github.com/cedricp/ddt4all/assets/168999811/775b5565-302e-471f-a4c9-2b99d4320a9d)

Après avoir accédé au menu d'écriture pour réinitialiser la dilution d'huile, je n'ai aucune réponse à la requête (réponse ELM vide) : 

![pas de reponse requete](https://github.com/cedricp/ddt4all/assets/168999811/0f5c7c37-1b54-48b5-b215-ac2e71920262)

Je ne sais pas si le soucis vient de mon ELM, du logiciel ou autres....? Quel boitier utilisez vous ?
Merci de m'avoir lu et pour votre aide !

Julien

**Comment by Furtif** [comment]
Je pense que votre hardware ou matériel OBD est mauvais, si vous n'avais pas des valeurs et si sa rame il n'est pas normal, comme écrit sur readme ddtall n'est pas compatible avec tout sort de matériel. il vous faut un que le soit.

**Reply by julien133** [reply]
Merci pour ton retour, j'ai commandé un nouveau ODB, je ferai un retour si ça a résolu le problème.

**Reply by Furtif** [reply]
Parfait, tiens nous au jus… mais si le matos est bon normalement tout va bien.

**Reply by julien133** [reply]
Salut, 
Après avoir changé l'ODB j'ai enfin pu avoir accès aux différents paramètres que je souhaitais donc le problème d'accès venait bien de l'ODB.

Je cherche maintenant à réduire la dilution d'huile car elle est lue à 11% ce qui est trop.
![image](https://github.com/cedricp/ddt4all/assets/168999811/3279b833-1a3e-4260-9fb9-4f2a24367d7e)

Dans la partie écriture IVP, j'ai essayé plein de valeur (ex :"0", "0.01", "0.001", "1") dans la case $211B : 
![image](https://github.com/cedricp/ddt4all/assets/168999811/b9625939-5d89-451d-a26d-6d7b20e3cbb6)

La requête est bien écrite et j'ai bien la réponse ELM mais quand je retourne dans l'écran de lecture, il ne se passe rien la valeur reste à 11%.
Quelqu'un a une idée ?

Merci d'avance

**Reply by Furtif** [reply]
Nikel si sa fonctionne!!! pour écrire sa faudra voir https://github.com/cedricp/ddt4all/discussions/1052#discussioncomment-8848667 voir comment il a fabriqué sa.

**Reply by Furtif** [reply]
Mark as ansewer si le probleme est reglé pour ce post et voir le post https://github.com/cedricp/ddt4all/discussions/1052#discussioncomment-8848667

**Reply by julien133** [reply]
Oui je lui ai posé la question, j'attends sa réponse

**Reply by Furtif** [reply]
okay click ```Mark as answer```

**Reply by Furtif** [reply, marked as answer]
la https://github.com/cedricp/ddt4all/discussions/1098#discussioncomment-9375544
